package.json

```diff
@@ -24,13 +24,16 @@
     "url": "https://github.com/nugit/actions-pr-changelog/issues"
   },
   "homepage": "https://github.com/nugit/actions-pr-changelog#readme",
+  "engines": {
+    "node": ">=14.17"
+  },
```

Comment:
Not sure if this is a breaking change. Because the action must run in Node14+ ?

Reply:
@moroine Hmmm, I'm not sure too. I can update it to `12`, but this repo needs `14` when developing. I think it is alright.

What do you think?

Reply:
I think you can mark a new major version no?

Reply:
Can also 👍🏻